server/athenian/api/openapi/openapi.yaml

```diff
@@ -1435,3 +1513,327 @@ components:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
+          - repository: github.com/athenianco/athenian-api
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
+      properties:
+        data:
+          items:
+            $ref: '#/components/schemas/CommitGroup'
+          type: array
+        include:
+          $ref: '#/components/schemas/IncludedNativeUsers'
+      required:
+      - data
+      - include
+      type: object
+    CommitGroup:
+      description: List of commits that were committed starting from a specific time
+        and ending at +granularity.
+      example:
+        date: 2000-01-23
+        commits:
+        - repository: github.com/athenianco/athenian-api
+          hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+          author:
+            login: github.com/gkwillie
+            name: Groundskeeper Willie
+            email: bot@athenian.co
+            timestamp: 2020-02-25T14:27:21Z
+          committer:
+            login: github.com/gkwillie
+            name: Groundskeeper Willie
+            email: bot@athenian.co
+            timestamp: 2020-02-25T14:27:21Z
+            message: 'Bump version: 0.0.42 → 0.0.43'
+            size_added: 1
+            size_removed: 1
+            files_changed: 1
+        - repository: github.com/athenianco/athenian-api
+          hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+          author:
+            login: github.com/gkwillie
+            name: Groundskeeper Willie
+            email: bot@athenian.co
+            timestamp: 2020-02-25T14:27:21Z
+          committer:
+            login: github.com/gkwillie
+            name: Groundskeeper Willie
+            email: bot@athenian.co
+            timestamp: 2020-02-25T14:27:21Z
+            message: 'Bump version: 0.0.42 → 0.0.43'
+            size_added: 1
+            size_removed: 1
+            files_changed: 1
+      properties:
+        date:
+          description: Commits were pushed beginning with this date. They were not
```

Comment:
_ditto_

Reply:
Hehe

Reply:
Let's keep it laser-focused for now and abstract later. It is hard to foresee what will be the future endpoints now. Better very specific now than refactoring crappy abstractions later.

Reply:
I don't see the benefit of keeping some docs that are not strictly true (because in the context of just `CommitGroup`, where this description is shown, the date interval and granularity has no meaning at all), and it can be easily outdated in the future as soon as other endpoint starts serving `CommitGroup`.

Reply:
The key word is as soon as. It is strictly true right now, and *as soon as* starts serving, I will address this issue. Don't invent problems and KISS. I can rename the model to be CommitGroupFor FilterCommitsRequest just to stop the intimidating, endless arguments with you :sweat:

Reply:
I have a strong opinion about this, but not strong enough to be considered as blocker this time. So yes, I do believe in your capacity for handling this in the best way. Let's continue as you're proposing.

Reply:
I mean, it's not needed to rename `FilterCommitRequest`, which is actually ok.

Reply:
Renamed to avoid confusion.